ic_routing_board_generation/board_generator/board_generator_v3_1_0_rb.py

```diff
@@ -0,0 +1,476 @@
+# constants has been moved to jumanji/environments/combinatorial/routing/constants.py
+# import jumanji.environments.combinatorial.routing.constants
+#from jumanji.environments.combinatorial.routing.constants import TARGET, HEAD, EMPTY
+#from jumanji.environments.combinatorial.routing.constants import SOURCE as WIRE
+HEAD, TARGET, WIRE, EMPTY = 4,3,2,0
+# Also available to import from constants OBSTACLE, NOOP, LEFT, LEFT, UP, RIGHT, DOWN
+from dataclasses import dataclass
+import numpy as np
+from copy import deepcopy
+import random
+from typing import List, Tuple
+#from jax.numpy import asarray # Currently jaxlib is not supported on windows.  This will have to be sorted.
+#from env_viewer import RoutingViewer # Currently jaxlib is not supported on windows.  This will have to be sorted.
+
+@dataclass#
+class Position:
+    # Class of 2D tuple of ints, indicating the size of an array or a 2D position or vector.
+    x: int
+    y: int
+
```

Comment:
position should only be in one file and then imported into each file - I think I put one in data model (I know you have import issues at the moment but this will need refactoring)

Reply:
As discussed, there will only be one file.  Old version are in the obsolete sub-folder.